lib/Target/OpenFhePke/OpenFhePkeEmitter.cpp

```diff
@@ -848,7 +914,13 @@ LogicalResult OpenFhePkeEmitter::printOperation(tensor::EmptyOp op) {
 
 LogicalResult OpenFhePkeEmitter::printOperation(tensor::ExtractOp op) {
   // const auto& v1 = in[0, 1];
-  emitAutoAssignPrefix(op.getResult());
+  if (isa<lwe::NewLWECiphertextType, lwe::LWECiphertextType>(
```

Comment:
how did you get the old lwe ciphertext types in the IR? was it meant to be a plaintext type?

Reply:
I added this mainly for completeness, though I admit it's probably not needed here.